plugins/default/lib/openshift.py

```diff
@@ -5,18 +5,31 @@
 def oc(*args):
     return check_output(("oc",) + args)
 
+
 def _get_running_pod_names(oc, project):
     # Manually filter Running pods because of a bug in `oc get`, see:
     # https://github.com/kubernetes/kubernetes/issues/29115
-    #return oc("-n", project, "get", "pods", "--show-all=false", "-o", "name")
-    pods = json.loads(oc("-n", project, "get", "pods", "--show-all=false", "-o", "json"))
-    return [p["metadata"]["name"] for p in pods if p["status"]["phase"] == "Running"]
+    # return oc("-n", project, "get", "pods", "--show-all=false", "-o", "name")
+    pods = json.loads(oc("-n", project, "get", "pods","--show-all=false", "-o", "json"))
+    return [p["metadata"]["name"] for p in pods["items"] if p["status"]["phase"] == "Running"]
+
 
 def get_running_pod_names(project):
-    return _get_running_pods(oc, project)
+    return _get_running_pod_names(oc, project)
+
 
 def _exec_in_pods(oc, project, pods, cmd):
-    return [oc("-n", project, "exec", name, "--", *cmd) for name in pods]
+    return [[name, oc("-n", project, "exec", name, "--", *cmd)] for name in pods]
```

Comment:
It was to make it easier to map the multi-line results back to the pod from which they came.  I guess I can use zip in the call to correlate the results back to the pod for which each belongs.  I.E.:
```
pods = openshift.get_running_pod_names(project)
    execs = openshift.exec_in_pods(project, pods, check_disk_cmd)
    for pod, lines in zip(pods, execs):
        results.extend(analize(pod, parse_df_lines(lines), warn, crit))
```